src/models/tools/geometry/jxg-table-link.ts

```diff
@@ -11,7 +11,6 @@ function createLinkedPoint(board: JXG.Board, parents: JXGCoordPair, props: any,
           clientType: "linkedPoint",
           fixed: true,
           fillColor: linkedPointColor,
-          clientFillColor: linkedPointColor,
           linkedTableId: tableId
         };
   const _props = { ...props, ...linkedProps };
```

Comment:
I guess colors get synced during point creation...would it be clearer to do it here too?

Reply:
My thinking is that most clients shouldn't need to know about `clientFillColor` or `syncClientColors()` -- they just set `fillColor` and the right thing happens automagically. That's the great benefit of your suggestion -- it encapsulates the functionality so that most of the client code doesn't need to know about it.